great-idea/css/index.css

```diff
@@ -63,4 +63,130 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+
+#container {
```

Comment:
I like the fact that you used a container here but I wouldn't advise putting a display flex on it. You will mainly want the container to just control things like width and centering of your sites content. You can also use it as a root div that controls font family, font colors and other cool things that you want to cascade down to the rest of your styles. If you put flex on this, that means the rest of your 2nd level divs will be forced to follow the flexbox rules and this can cause issues with placement and really limit what you can do.